src/data/items.txt

```diff
@@ -7993,10 +7993,10 @@
 7965	Holy MacGuffin	534962786	macguffin.gif	none	q	0
 7966	Ka coin	826932303	kacoin.gif	none	q	0
 7967	World's Best Adventurer sash	584125368	bestsash.gif	accessory	q	0	World's Best Adventurer sashes
-7968	topiary nugglet	491943392	nugglet.gif	none	t	0
+7968	topiary nugglet	491943392	nugglet.gif	reusable	t	0
 7969	beehive	905003610	beehive.gif	none, combat reusable	q	0
 7970	electric boning knife	580217520	elecbone.gif	none, combat reusable	q	0	electric boning knives
-7971	Aggressive Carrot	839021006	carrot.gif	none, curse	t	0
+7971	Aggressive Carrot	839021006	carrot.gif	usable	t	0
```

Comment:
Is it not still a "curse" item? I.e., you need to give it a user name to afflict.

Reply:
trying to use it in `curse.php` doesn't work. It sends you to a choice adventure.